### PR TITLE
Update local changes to gradle

### PR DESCRIPTION
**Pull request by brandonhoughton**
The current gradle version is too old for Intellij, this updates it and enables using the setup script without version modification

### CODE REVIEW

File: minerl/Malmo/Minecraft/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14-bin.zip
```
Comment (jietang) [inline minerl/Malmo/Minecraft/gradle/wrapper/gradle-wrapper.properties:6]: is this backslash important? I have it in my gradle-wrapper.properties, based on your slack message https://openai.slack.com/archives/C01RJA7BEPP/p1616105094042400

File: minerl/Malmo/Minecraft/build.gradle [hunk 2]
```diff
@@ -346,7 +346,38 @@ task copyModToServer(type: Copy) {
 // Interesting. We've packaged forge gradle into the full jar and use gradle inherently??
 shadowJar {
     classifier = 'fat'
-    configurations = [project.configurations.all]
+    configurations = [
```
Comment (jietang) [inline minerl/Malmo/Minecraft/build.gradle:349]: where does this list of configurations come from?

**Review by jietang — APPROVED** [review]
don't fully understand the mechanics here, but if it works locally and on CI I'm all for merging

File: minerl/Malmo/Minecraft/build.gradle (hunk 2 above)
Comment (brandonhoughton) [reply minerl/Malmo/Minecraft/build.gradle]: I updated this to be more clean

File: minerl/Malmo/Minecraft/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (brandonhoughton) [reply minerl/Malmo/Minecraft/gradle/wrapper/gradle-wrapper.properties:6]: I'm going to go with this is not important unless it breaks something somewhere =) which it does not seem to do for ci

**Pull request merged**